otherlibs/unix/unixLabels.mli

```diff
@@ -1386,24 +1390,24 @@ type group_entry = Unix.group_entry =
   }
 (** Structure of entries in the [groups] database. *)
```

Comment:
`sleep` and `sleepf` are kinda unsafe, usually they use `nanosleep` or `select` but it can fall back to actual `sleep` which is unsafe in the presence of `SIGALRM`.... I'm not sure about this one, somewhat tempted to just pretend it's fine.